components/popover/popover-mixin.js

```diff
@@ -124,20 +127,26 @@ export const PopoverMixin = superclass => class extends superclass {
 	}
 
 	close() {
-		this._opened = false;
-		return this.updateComplete;
+		if (!this._opened) return Promise.resolve();
+		return new Promise(resolve => {
+			this.addEventListener('d2l-popover-close', resolve, { once: true });
+			this._opened = false;
+		});
 	}
 
 	configure(properties) {
 		this._noAutoClose = properties?.noAutoClose ?? false;
 		this._noAutoFocus = properties?.noAutoFocus ?? false;
-		this._opened = properties?.opened ?? false;
```

Comment:
I removed this because it was causing a problems in one of the test cases where the popover element is created and initialized in the open state before being attached to the DOM.
```javascript
const popover = document.createElement('my-popover');
popover.open();
document.body.appendChild(popover);
```
Basically, `open()` would open the popover (`_opened = true`), but when the element is attached to the DOM, the sub-class would call `configure()`, which would set it back to `false`. It's an edge case that might not come up, but it's safer and less confusing if there is only one way to set the mixin's opened state.